### PR TITLE
Sn test

### PR DESCRIPTION
**Pull request by narwalsandeep**
test cases need reviews - @alengel @pascal-lightning

### CODE REVIEW

File: src/com/rest/utils/DatabaseAccess.java [hunk 1]
```diff
@@ -721,6 +721,35 @@ public boolean updateUser(String key, String password,
 		return true;
 	}
 
+	public boolean resetPassword(String email){
```
Comment (alengel) [inline src/com/rest/utils/DatabaseAccess.java:724]: A comment here please.

File: src/com/user/test/DatabaseAccessTest.java [hunk 2]
```diff
@@ -388,9 +388,344 @@ public void testCheckInUnsuccessful() throws ArgumentMissingException, InvalidKe
 		assertEquals (expected, result);
 	}
 
+	
+	/*
+	* Testing StoreNewVote
```
Comment (alengel) [inline src/com/user/test/DatabaseAccessTest.java:393]: Same as the other PR, please put more descriptive comments above each function.

File: src/com/user/test/DatabaseAccessTest.java [hunk 3]
```diff
@@ -388,9 +388,344 @@ public void testCheckInUnsuccessful() throws ArgumentMissingException, InvalidKe
...
+		String firstName = "Karolina";
+		String lastName = "Schliski";
+		String password = "Hi98786";
+		
+		//dbAccess.registerNewUser(email, password, firstName, lastName, null);
+		
+		//UserData userLoggedIn = dbAccess.loginUser(email, password);
+		
+		//String loginKey = userLoggedIn.getLoginKey();
+
```
Comment (alengel) [inline src/com/user/test/DatabaseAccessTest.java:553]: No commented out code, please remove.

File: src/com/user/test/DatabaseAccessTest.java [hunk 4]
```diff
@@ -388,9 +388,344 @@ public void testCheckInUnsuccessful() throws ArgumentMissingException, InvalidKe
...
+		String email = "test@web.de";
+		String firstName = "Karolina";
+		String lastName = "Schliski";
+		String password = "Hi98786";
+		
+		//dbAccess.registerNewUser(email, password, firstName, lastName, null);
+		
+		//UserData userLoggedIn = dbAccess.loginUser(email, password);
+		
+		//String loginKey = userLoggedIn.getLoginKey();
```
Comment (alengel) [inline src/com/user/test/DatabaseAccessTest.java:574]: No commented out code, please remove.

File: src/com/user/test/DatabaseAccessTest.java [hunk 5]
```diff
@@ -388,9 +388,344 @@ public void testCheckInUnsuccessful() throws ArgumentMissingException, InvalidKe
...
```
Comment (alengel) [inline src/com/user/test/DatabaseAccessTest.java:699]: No commented out code, please remove.

File: src/com/user/test/DatabaseAccessTest.java [hunk 6]
```diff
@@ -388,9 +388,344 @@ public void testCheckInUnsuccessful() throws ArgumentMissingException, InvalidKe
...
```
Comment (alengel) [inline src/com/user/test/DatabaseAccessTest.java:721]: No commented out code, please remove.

**Comment by narwalsandeep** [comment]
im on ec2 and php an still lot of work, will take time, plz have someone to add comments whosoever can do quick.

**Pull request closed without merging**